test/index.html

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
+<html>
+  <head></head>
+  <body>
+    <h1>Hello World!</h1>
+  </body>
+</html>
```

Comment:
Following [UNIX conventions](https://robots.thoughtbot.com/no-newline-at-end-of-file), can we add a newline at the end of this file?